### PR TITLE
Input mask transform api docs

### PR DESCRIPTION
**Pull request by atk**

### CODE REVIEW

File: packages/input-mask/README.md [hunk 1]
```diff
@@ -106,7 +106,18 @@ const { field, form } = useField(props.name);
...
 
-To use the mask handler with [Modular Forms](https://modularforms.dev/), you may need to overwrite the re-validation handler:
+To use the mask handler with [Modular Forms](https://modularforms.dev/), from version 0.13.1, you can use its [Transform inputs API](https://modularforms.dev/solid/guides/transform-inputs):
+
+```tsx
+<Field
+  name="ccard"
+  transform={toCustom((_value, event) => ccmask(event), { on: "input" })}
+>
+  {(field) => <input {...field.props} />}
```
Comment (fabian-hiller) [inline packages/input-mask/README.md:116]: `{(field, props) => <input {...props} value={field.value} />}`

File: packages/input-mask/README.md [hunk 2]
```diff
@@ -106,7 +106,18 @@ const { field, form } = useField(props.name);
 <input onBlur={form.handleBlur} onInput={ev => (inputMask(ev), form.handleChange(ev))} />;
 ```
 
-To use the mask handler with [Modular Forms](https://modularforms.dev/), you may need to overwrite the re-validation handler:
+To use the mask handler with [Modular Forms](https://modularforms.dev/), from version 0.13.1, you can use its [Transform inputs API](https://modularforms.dev/solid/guides/transform-inputs):
+
+```tsx
+<Field
+  name="ccard"
+  transform={toCustom((_value, event) => ccmask(event), { on: "input" })}
```
Comment (fabian-hiller) [inline packages/input-mask/README.md:114]: `transform={toCustom((_, event) => ccmask(event), { on: "input" })}`

File: packages/input-mask/README.md [hunk 3]
```diff
@@ -106,7 +106,18 @@ const { field, form } = useField(props.name);
...
+>
+  {(field) => <input {...field.props} />}
+</Field>
+```
+
+For older versions than 0.13.x which don't support this helpful API, you may need to overwrite the re-validation handler:
 
 ```tsx
 <Field
```
Comment (fabian-hiller) [inline packages/input-mask/README.md:123]: `<Field of={…} name={…}>`

**Review by fabian-hiller — CHANGES_REQUESTED** [review]

**Comment by thetarnav** [comment]
neovim git moment

image

**Pull request merged**